java-checks/src/main/java/org/sonar/java/symexecengine/ExecutionState.java

```diff
@@ -36,18 +37,41 @@
 
 public class ExecutionState {
 
-  final ExecutionState parent;
-  private SetMultimap<Symbol, SymbolicValue> reachableValues = HashMultimap.create();
+  public final ExecutionState parent;
+
+  /*
+   * definitelyReachableValues and potentiallyReachableValues contains the set of possible values from a given symbol.
```

Comment:
That part of documentation should be correctly formatted and moved to the javadoc of the class, as it explains how react the different sets of values. Adding the unreachable values should help too.